shell.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs ? import <nixpkgs> { } }:
+
+pkgs.mkShell rec {
+  name = "parca-agent";
+
+  packages = with pkgs; [
+    clang
+    gnumake
+    go_1_18
+    kubectl
+    libbpf
+    libelf
+    llvm
+    minikube
+    nodejs-16_x
```

Comment:
I think we don't need nodejs in this repo. I was that it's mentioned in the contributing.md above, we should also remove it. I think it's a residue from parca repo.

Reply:
It is probably because there are steps to build `parca` here:

https://github.com/parca-dev/parca-agent/blob/51aee7a5fb288d9a1db111ebfc395e213c15e4a4/CONTRIBUTING.md#L77-L79

Should we remove everything specific to parca and link its `CONTRIBUTING.md` instead?

Reply:
Yes, let's do that. Do you want to handle it? Otherwise, I'd be happy to do it.

Reply:
Done!